content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+    kubectl apply --context=${CTX_FRED} -f -
+{{< /text >}}
+
+Since the clusters are on separate networks, we need to expose all services
+(*.local) on the east-west gateway in both clusters. While this gateway is
+public on the Internet, services behind it can only be accessed by services
+with a trusted mTLS certificate and workload ID, just as if they were on the
+same network.
+
+{{< text bash >}}
+$ kubectl --context=${CTX_FRED} apply -n istio-system -f \
+    samples/multicluster/expose-services.yaml
+{{< /text >}}
+
+### Configure `BARNEY` as a primary with services exposed
+
+Install Istio on `BARNEY`:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_BARNEY} -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: BARNEY
+      network: EAST
+      meshNetworks:
+        WEST:
+          endpoints:
+          - fromRegistry: FRED
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+        EAST:
+          endpoints:
+          - fromRegistry: BARNEY
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+As we did with `FRED` above, install a gateway in `BARNEY` that is dedicated
+to east-west traffic and expose user services.
+
+{{< text bash >}}
+$ CLUSTER=BARNEY NETWORK=EAST \
+    samples/multicluster/gen-eastwest-gateway.sh | \
+    kubectl apply --context=${CTX_BARNEY} -f -
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl --context=${CTX_BARNEY} apply -n istio-system -f \
+    samples/multicluster/expose-services.yaml
+{{< /text >}}
+
+### Enable Endpoint Discovery for `FRED` and `BARNEY`
+
+Install a remote secret in `BARNEY` that provides access to `FRED`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_FRED} \
+  --name=FRED | \
+  kubectl apply -f - --context=${CTX_BARNEY}
+{{< /text >}}
+
+Install a remote secret in `FRED` that provides access to `BARNEY`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_BARNEY} \
+  --name=BARNEY | \
+  kubectl apply -f - --context=${CTX_FRED}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Primary-Remote" category-value="primary-remote" >}}
+
+The following steps will install the Istio control plane on `FRED` (the
+{{< gloss >}}primary cluster{{< /gloss >}}) and configure `BARNEY` (the
+{{< gloss >}}remote cluster{{< /gloss >}}) to use the control plane in `FRED`. Both
+clusters reside on the `EAST` network.
+
+`FRED` will be configured to observe the API Servers in both clusters for
+endpoints. In this way, the control plane will be able to provide service
+discovery for workloads in both clusters. Service workloads across cluster
+boundaries communicate indirectly, via dedicated
```

Comment:
This is wrong, they're going direct, it is only the istiod connection that goes through a gateway.

We should talk here about how control plane traffic goes through a gateway but data plane traffic goes direct. The control plane is indirect because of bootstrapping, we need a stable IP address for istiod and we use an ingress gateway to get that.